magmi/inc/magmi_engine.php

```diff
@@ -513,7 +513,26 @@ public function connectToMagento()
                 $user = $default_setup->username;
                 $pass = $default_setup->password;
                 $port = $default_setup->port;
-            } else {
+            }else if($conn == 'envphp'){ //berto - scraping env.php
```

Comment:
please fix code formatting and remove unnecessary comments

Reply:
Ok done. Check know please